lib/github/middleware.js

```diff
@@ -80,12 +80,15 @@ module.exports = function middleware (callback) {
     context.sentry.setTag('transaction', `webhook:${context.name}.${context.payload.action}`)
     for (let subscription of subscriptions) {
       const jiraHost = subscription.jiraHost
-
       context.sentry.setTag('jiraHost', jiraHost)
       context.sentry.setTag('gitHubInstallationId', gitHubInstallationId)
       context.sentry.setUser({ jiraHost, gitHubInstallationId })
-
       context.log = context.log.child({ jiraHost })
+      if (context.timedout) {
+        Sentry.captureException(new Error('Timed out jira middleware iterating subscriptions'));
```

Comment:
This isn't the greatest, but I'd like to send this to sentry too. This to me is an actionable error for us to work on.

I think there is value in abstracting this later if/when we start reusing this pattern more.